PYME/LMVis/Extras/animation.py

```diff
@@ -57,7 +57,7 @@ def __init__(self, parent_panel, **kwargs):
         self.next_view_id = 0
 
     def create_buttons(self, vertical_sizer):
-        grid_sizer = wx.GridSizer(3, 3)
+        grid_sizer = wx.GridSizer(3, 3, 3)

```

Comment:
should be `wx.GridSizer(3,3,0,0)` or `wx.GridSizer(rows=3, cols=3, vgap=0, hgap=0)`to match both wx3 and wx4 signatures (I half expect the wx4 signature to change as it's currently pretty broken).

Reply:
The wx4 signature certainly accepts both (https://wxpython.org/Phoenix/docs/html/wx.GridSizer.html). No idea about wx3.